api/tests/unit/application/target-profiles/index_test.js

```diff
@@ -160,15 +165,61 @@ describe('Integration | Application | Target Profiles | Routes', function () {
       const httpTestServer = new HttpTestServer();
       await httpTestServer.register(moduleUnderTest);
 
-      const method = 'GET';
-      const url = '/api/admin/target-profiles/azerty';
-
       // when
-      const response = await httpTestServer.request(method, url);
+      const response = await httpTestServer.request(method, '/api/admin/target-profiles/azerty');
```

Comment:
On a utilisé let url et là c'était l'occasion de lui réaffecter une valeur et justifier le beforeEach.
Je propose soit de remettre comme c'était, soit d'enlever aussi le beforeEach et de définir les valeurs d'url et method via des const

Autre remarque, le test qui s'appelle 'should resolve with correct id' ligne 147  peut être supprimé car ça ne teste pas plus que 'should return a response with an HTTP status code 200 when user has role "SUPER_ADMIN", "SUPPORT", "METIER"' (voir ça teste moins car on a pas utilisé le withArgs dans le stub)